packages/pwa-kit-create-app/scripts/build.js

```diff
@@ -37,22 +39,25 @@ const main = () => {
 
     const tmpDir = mkdtempSync()
     const checkoutDir = p.join(tmpDir, 'mobify-platform-sdks')
+    const packageDir = p.join(checkoutDir, 'packages')
 
     sh.exec(
         `git clone --config core.longpaths=true --single-branch ` +
             `--depth=1 file://${monorepoRoot} ${checkoutDir}`
     )
 
     return Promise.all(
-        pkgNames.map((pkgName) =>
-            tar.c(
+        pkgNames.map((pkgName) => {
+            sh.mv(p.join(packageDir, `${TEMPLATE_PREFIX}${pkgName}`), p.join(packageDir, pkgName))
```

Comment:
Does the `sh.mv` command have any risk of unwanted side-effects where running the generator twice from local (is that a "thing"?) as opposed to via `npx` might fail the second time because the files got moved?

Reply:
There shouldn't be any risk here.. all the work is happening in the a temp folder that is ephemeral and does not exist from one generation to the next. The reason I changed this to a `mv` was because when you download via the pack command the content of the tarball is a folder named package, and that isn't true for the "bundle" tarballs we create.